src/utils/intl/index.ts

```diff
@@ -0,0 +1,24 @@
+import i18next from "i18next";
+
+ i18next.init({
+  lng: "en", // if you're using a language detector, do not define the lng option
+  debug: true,
+  resources: {
+    en: {
+      translation: {
```

Comment:
In the next changes, It would be nice and more maintainable to have these in their own files, even better if broken module by module to reduce the size of files.

Reply:
@brianmuks take note of this but for next PR.